src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,251 @@
+package Duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import Duke.Exceptions.*;
```

Comment:
Try to avoid wildcard `*` import statements for better readability purposes. If possible, you can list out the relevant classes you need. I understand that you used all the classes in those packages but for others who are reading your code for the first time, they may be unaware of this.